### PR TITLE
Added installation prerequisites with link to ADB install doc

### PR DESCRIPTION
**Pull request by thatdocslady**
Related to issue 450 in the ADB repository: https://github.com/projectatomic/adb-atomic-developer-bundle/issues/450

This PR adds a prerequisites step to the installation, with a link to the main ADB install instructions (to avoid documenting the information in multiple places).

### CODE REVIEW

**Comment by hferentschik** [comment]
please test

File: README.adoc [hunk 1]
```diff
@@ -24,6 +24,11 @@ toc::[]
 [[img-build-status]]
 image::https://ci.centos.org/buildStatus/icon?job=vagrant-service-manager[link="https://ci.centos.org/job/vagrant-service-manager"]
 
+== Prerequisites
+
+Follow the [ADB installation instructions](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst#installing-the-atomic-developer-bundle)
```
Comment (bexelbie) [inline README.adoc]: Please correct the link to use asciidoc and not markdown format.

File: README.adoc [hunk 2]
```diff
@@ -24,6 +24,11 @@ toc::[]
 [[img-build-status]]
 image::https://ci.centos.org/buildStatus/icon?job=vagrant-service-manager[link="https://ci.centos.org/job/vagrant-service-manager"]
 
+== Prerequisites
```
Comment (hferentschik) [inline README.adoc:27]: Is this really needed? By the time you deal with the vagrant-service-manager you already have a VM setup, right? Nothing against being explicit, but this feels too much too me.

@bexelbie, WDYT?

Comment (thatdocslady) [inline README.adoc:27]: @hferentschik the main reason for adding this section is to allow users to jump "upwards" towards the main ADB install instructions and review any limitations or OS-specific prerequisites. In the PR description you'll see there's a related PR in the ADB repo for this.

File: README.adoc (hunk 1 above)
Comment (thatdocslady) [inline README.adoc]: @bexelbie how I long for a facepalm emoji ;)

File: README.adoc (hunk 2 above)
Comment (hferentschik) [inline README.adoc:27]: Except the "Prerequisites" for this plugin is a Vagrant install. Whether an install of the plugin makes any sense without using a ADB/CDK VM is a different question, but the prerequisite here is Vagrant.

Comment (hferentschik) [inline README.adoc:27]: There is already a link to ADB in the top section, but maybe we can find yet another way to link back.

**Comment by hferentschik** [comment]
@thatdocslady I suggest something like in this pull request #359. WDYT? Also created an issue for it - #358

**Comment by hferentschik** [comment]
Superseded by pull request  #359

**Pull request closed without merging**